graders/python/python_autograder/pl_helpers.py

```diff
@@ -22,6 +20,9 @@ class GradingSkipped(Exception):  # noqa: N818
 
 
 def extract_ipynb_contents(f, ipynb_key):
+    from IPython.core.interactiveshell import InteractiveShell
+    from nbformat import read
```

Comment:
This is an unrelated issue; but these functions are only defined in the context of a ipynb notebook.